src/jarabe/journal/listview.py

```diff
@@ -711,6 +716,17 @@ def __drag_data_get_cb(self, widget, context, selection, info, time):
         model = self.tree_view.get_model()
         model.do_drag_data_get(path, selection)
 
+    def __button_press_event_cb(self, widget, event):
+        try:
+            path = self.tree_view.get_path_at_pos(event.x, event.y)[0]
+        except:
```

Comment:
No, the problem is that your *except* does not have the name of an exception, so it will hide all exceptions, including coding errors.